roles/kafka/tasks/config.yml

```diff
@@ -48,6 +50,34 @@
     - kafka:create_data_directory
 
 
+- name: "Recursively remove  /etc/kafka   directory"
```

Comment:
Yes you are right, but I do it because of :
* https://github.com/Tas-sos/argo-ansible/blob/e89f408cbdddf9f5a422914687274386769d97fa/roles/zookeeper/tasks/zookeeper.yml#L130-L148
and if we don't remove this directory, the `mv` command has this problem : https://pastebin.com/raw/jz2QcbUr

In addition, we want **only a symlink** from `/usr/lib/zookeeper/conf/` to `/etc/zookeeper/conf`. The problem isn't the permissions. That's why it's a bit confusing..

Reply:
:+1: